content/en/monitors/faq/anomaly-monitor.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Anomaly Monitor FAQ
+kind: faq
+---
+
+### Should I use anomaly detection for everything?
+
+No. Anomaly detection is designed to assist with visualizing and monitoring metrics that have predictable patterns. For example, `my_site.page_views{*}` might be driven by user traffic and thus vary predictably by time of day and day of week. If your metric does not have a repeated or predictable pattern, then a simple chart overlay or threshold alert might be better.
+
+**Note**: Anomaly detection requires historical data to make good predictions. If you have only been collecting a metric for a few hours or days, anomaly detection won't be useful.
+
+### Why can't I use anomaly detection over groups in the dashboard?
+
+Adding many separate timeseries in a single graph can lead to [spaghettification][1], and the problem gets worse when the anomaly detection visualization is added:
+
+{{< img src="monitors/monitor_types/anomaly/spaghetti.png" alt="spaghetti" responsive="true" style="width:80%;">}}
+
+However, it is possible to add multiple series to a single graph one at a time. The gray band only shows up on mouseover:
+
+{{< img src="monitors/monitor_types/anomaly/anomaly_multilines.png" alt="anomaly multi lines" responsive="true" style="width:80%;" >}}
+
+### Do past anomalies affect the current predictions?
+
+All the algorithms except `basic` use extensive amounts of historical data so they are robust to most anomalies. In the first graph, the gray band stays around 400K even after the metric has dropped to 0.
+
+{{< img src="monitors/monitor_types/anomaly/anomalous_history.png" alt="anomalous_history" responsive="true" style="width:80%;">}}
+
+The second graph shows the same metric a day later. Even though it uses the previous day in the calculation of the band, it is unaffected by the anomaly that occurred previously.
+
+{{< img src="monitors/monitor_types/anomaly/no_effect.png" alt="no effect" responsive="true" style="width:80%;">}}
+
+### Why does an anomaly "disappear" when I zoom in?
+
+At different zoom levels, the same query can result in timeseries with very different characteristics. When looking at longer time periods, each point represents the aggregate of many more-granular points. Therefore, each of these aggregate points may hide noise observed in the more granular points. For example, charts that show one week often appear smoother (less noisy) than charts that show just 10 minutes.
+
+The width of the gray band is partly based on the noisiness of the timeseries in the plot. The band must be wide enough that ordinary noise is mostly inside the band and doesn't appear as anomalous. Unfortunately, when the band is wide enough to include ordinary noise, it might also be wide enough to hide some anomalies, especially when viewing short time windows.
```

Comment:
```suggestion
Setting the width of the gray band correctly is key to accurate anomaly monitoring. The band must be wide enough that ordinary noise is mostly inside the band and doesn't appear as anomalous. Unfortunately, when the band is wide enough to include ordinary noise, it might also be wide enough to hide some anomalies, especially when viewing short time windows.
```

Reply:
The user cannot set the width of the gray band. I will reword this a little.